.dialyzer_ignore.exs

```diff
@@ -0,0 +1,3 @@
+[
+    { "lib/phoenix/router.ex", :pattern_match, 402 }
```

Comment:
@lcpojr to resolve the dialyzer warning about phoenix, this is the solution i've found -> https://elixirforum.com/t/dialyzer-error-when-using-phoenix-router-match-5/32121/3